src/1b_prep_aca.jl

```diff
@@ -96,4 +96,61 @@ region_features[!, :geometry] = Vector{AG.IGeometry}(AG.forceto.(region_features
         src_aca_bathy = nothing
         GC.gc()
     end
+
+    if !isfile(joinpath(ACA_OUTPUT_DIR, "$(reg)_waves_Hs.tif"))
+        target_waves_Hs_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, "Hs", reg)))
+        target_waves_Hs = Raster(target_waves_Hs_path, key=:Hs90, crs=crs(src_aca_bathy), mappedcrs=EPSG(4326), lazy=true)
+
+        # Extend bounds of wave data to match bathymetry if needed
+        if size(src_aca_bathy) !== size(target_waves_Hs)
+            target_waves_Hs = extend(crop(target_waves_Hs; to=src_aca_bathy); to=AG.extent(src_aca_bathy))
+            @assert size(src_aca_bathy) == size(target_waves_Hs)
+
+            replace_missing!(target_waves_Hs, -9999.0)
```

Comment:
I think you will need to replace -9999.0 with the missing val for aca_bathy. (0 ?)